packages/inscription/src/inscription/inscription-ui.tsx

```diff
@@ -73,28 +74,25 @@ export class InscriptionUi extends AbstractUIExtension implements IActionHandler
 
   protected initializeContents(containerElement: HTMLElement) {
     this.initConnection();
-    this.changeUiVisiblitiy();
+    this.changeUiVisiblitiy(false);
     IvyScriptLanguage.startWebSocketClient(this.webSocketAddress + 'ivy-script-lsp');
     this.queryClient = initQueryClient();
     this.inscriptionClient = this.startInscriptionClient();
-
     this.root = createRoot(containerElement);
-    this.updateInscriptionUi();
```

Comment:
This caused the EnableInscriptionAction also already load the inscription for the process, which also needs some time. Now this is postponed to the first selection. So I hope this also increases the first startup speed by some ms...